financial-connections/src/main/java/com/stripe/android/financialconnections/features/common/SharedPartnerAuth.kt

```diff
@@ -136,41 +133,27 @@ private fun SharedPartnerAuthContent(
     onCancelClick: () -> Unit,
     onCloseFromErrorClick: (Throwable) -> Unit
 ) {
-    val scope = rememberCoroutineScope()
-    FinancialConnectionsModalBottomSheetLayout(
-        sheetState = bottomSheetState,
-        sheetContent = {
-            state.dataAccess?.let {
-                DataAccessBottomSheetContent(
-                    dataDialog = it,
-                    onConfirmModalClick = { scope.launch { bottomSheetState.hide() } },
-                    onClickableTextClick = onClickableTextClick
-                )
-            } ?: Spacer(modifier = Modifier.size(16.dp))
-        },
-        content = {
-            SharedPartnerAuthBody(
-                state = state,
-                onCloseClick = onCloseClick,
-                onSelectAnotherBank = onSelectAnotherBank,
-                onEnterDetailsManually = onEnterDetailsManually,
-                onCloseFromErrorClick = onCloseFromErrorClick,
-                onClickableTextClick = onClickableTextClick,
-                onCancelClick = onCancelClick,
-                onContinueClick = onContinueClick,
-            )
-        }
+    SharedPartnerAuthBody(
+        inModal = inModal,
+        state = state,
+        onCloseClick = onCloseClick,
+        onSelectAnotherBank = onSelectAnotherBank,
+        onEnterDetailsManually = onEnterDetailsManually,
+        onCloseFromErrorClick = onCloseFromErrorClick,
+        onClickableTextClick = onClickableTextClick,
+        onCancelClick = onCancelClick,
+        onContinueClick = onContinueClick,
     )
 }
 
 @Composable
 @Suppress("MagicNumber")
-private fun SharedPartnerLoading() {
+private fun SharedPartnerLoading(inModal: Boolean) {
```

Comment:
updated the loading shimmer to look more like the new content + adapt to the modal or full screen display style accordingly.